roles/collect/defaults/main.yml

```diff
@@ -1,2 +1,17 @@
 ---
 # defaults file for collect
+collect_endpoint_scheme: 'http'
+collect_endpoint_host: 'localhost'
+collect_endpoint_port: '8001'
+collect_endpoint_path: 'api/v1/namespaces/metering/services/https:reporting-operator:http/proxy/api/v1/reports/get'
```

Comment:
Talked to Andrew and looks like I need to consume the namespace for metering too in the setup. So this endpoint path will be based on that input value I will save in the config file, `ocp_metering_namespace`.

Not a blocker,  just some info since we are iterating.

Reply:
_Note:_
The above setup works if we can do an `oc login` with token and then `oc proxy` and hit the endpoint. Question is that doable with existing ansible calls?

Reply:
I'm not seeing `oc login` or `oc proxy` in the lib_openshift stuff. So, I think we'll have to resort to `command` here.